sh/install-tests.sh

```diff
@@ -3,37 +3,21 @@
 # bash script mode
 set -euo pipefail
 
-trap 'kill $(jobs -p)' EXIT
-
 set -a
 source .env || { echo '[install-tests.sh] could not load .env, using variables from environment.' ; }
 set +a
 
-./sh/build-init-database.sh
+./sh/build-init-database.sh || { echo 'failed to build DB' ; exit 1; }
```

Comment:
Nice.  I had forgotten that you could to this scripts as well as individual commands.